rpcserverhandlers_test.go

```diff
@@ -677,81 +690,345 @@ type rpcTest struct {
+	testChainParams.Name = "mainnet"
+	testChainParams.Deployments = map[uint32][]chaincfg.ConsensusDeployment{
+		7: {{
+			Vote: chaincfg.Vote{
+				Id:          chaincfg.VoteIDHeaderCommitments,
+				Description: "Enable header commitments as defined in DCP0005",
+				Mask:        0x0006, // Bits 1 and 2
+				Choices: []chaincfg.Choice{{
+					Id:          "abstain",
+					Description: "abstain voting for change",
+					Bits:        0x0000,
+					IsAbstain:   true,
+					IsNo:        false,
+				}, {
+					Id:          "no",
+					Description: "keep the existing consensus rules",
+					Bits:        0x0002, // Bit 1
+					IsAbstain:   false,
+					IsNo:        true,
+				}, {
+					Id:          "yes",
+					Description: "change to the new consensus rules",
+					Bits:        0x0004, // Bit 2
+					IsAbstain:   false,
+					IsNo:        false,
+				}},
+			},
+			StartTime:  1567641600, // Sep 5th, 2019
+			ExpireTime: 1599264000, // Sep 5th, 2020
+		}},
+	}
+	testChainParams.PowLimitBits = 0x1d00ffff
+	return testChainParams
+}()
+
+// defaultMockRPCChain provides a default mock chain to be used throughout
+// the tests.  Tests can override these defaults by calling defaultMockRPCChain,
+// updating fields as necessary on the returned *testRPCChain, and then setting
+// rpcTest.mockChain as that *testRPCChain.
+func defaultMockRPCChain() *testRPCChain {
+	// Define variables related to block432100 to be used as default values for the
+	// mock chain.
+	blk := dcrutil.NewBlock(&block432100)
+	blkHeader := block432100.Header
+	blkHash := blk.Hash()
+	blkHeight := blk.Height()
+	chainWork, _ := new(big.Int).SetString("0e805fb85284503581c57c", 16)
+	filter, _ := gcs.FromBytesV2(20, 1<<20, nil)
+
+	return &testRPCChain{
+		bestSnapshot: &blockchain.BestState{
+			Hash:           *blkHash,
+			PrevHash:       blkHeader.PrevBlock,
+			Height:         blkHeight,
+			Bits:           blkHeader.Bits,
+			NextPoolSize:   41135,
+			NextStakeDiff:  14428162590,
+			BlockSize:      uint64(blkHeader.Size),
+			NumTxns:        7,
+			TotalTxns:      7478697,
+			MedianTime:     time.Unix(1584246683, 0), // 2020-03-15 04:31:23 UTC
+			TotalSubsidy:   1122503888072909,
+			NextFinalState: [6]byte{0xdc, 0x2a, 0x4f, 0x6e, 0x60, 0xb3},
+		},
+		blockByHash:                     blk,
+		blockByHeight:                   blk,
+		blockHashByHeight:               blkHash,
+		blockHeightByHash:               blkHeight,
+		calcNextRequiredStakeDifficulty: 14428162590,
+		calcWantHeight:                  431487,
+		chainTips: []blockchain.ChainTipInfo{{
+			Height:    blkHeight,
+			Hash:      *blkHash,
+			BranchLen: 500,
+			Status:    "active",
+		}},
+		chainWork: chainWork,
+		convertUtxosToMinimalOutputs: []*stake.MinimalOutput{{
+			PkScript: hexToBytes("BAA914780239EA1231BA67B0C5B82E786B51E21072522187"),
+			Value:    100000000,
+			Version:  0,
+		}, {
+			PkScript: hexToBytes("6A1E355C96F48612D57509140E9A049981D5F9970F945C770D00000000000058"),
```

Comment:
I know it's nit picking, but please make these lowercase hex for consistency with the rest of the code.

Reply:
Agreed, it should be consistent, updated